resources/prosody-plugins/mod_fmuc.lua

```diff
@@ -428,10 +428,12 @@ module:hook_global('stats-update', function ()
     for room in prosody.hosts[module.host].modules.muc.each_room() do
         rooms_count = rooms_count + 1;
         for _, o in room:each_occupant() do
-            if jid.host(o.bare_jid) == local_domain then
-                visitors_count = visitors_count + 1;
-            else
-                participants_count = participants_count + 1;
+            if not is_admin(o.bare_jid) then
```

Comment:
There's code below which accounts for jicofo, now it's accounted twice.

Reply:
ups